Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -68,10 +68,19 @@ open class CarPlayNavigationViewController: UIViewController {
      */
     public private(set) var styleManager: StyleManager?
     
+    /**
+     Allows to control highlighting of the destination building on arrival. By default destination buildings will not be highlighted.
```

Comment:
The documentation looks like it is for a different property.

Reply:
This is original name of the property we came up with to control building highlighting.